en/starter/generator/command_line_tools.md

```diff
@@ -0,0 +1,154 @@
+---
+### TRANSLATION INSTRUCTIONS FOR THIS SECTION:
+### TRANSLATE THE VALUE OF THE title ATTRIBUTE AND UPDATE THE VALUE OF THE lang ATTRIBUTE.
+### DO NOT CHANGE ANY OTHER TEXT.
+layout: page
+title: Command-line tools
+menu: starter
+lang: en
+redirect_from: "/starter/generator/command_line_tools.html"
+### END HEADER BLOCK - BEGIN GENERAL TRANSLATION
+---
+
+<div class="titleBlock">
+	<h1>Command-line tools</h1>
+	<p>This article summarizes Swift Server Generator CLI tool commands.</p>
+</div>
+
+Two command-line tools are available for Swift Server Generator:
+
+-   [Yeoman](http://yeoman.io/) command-line tools: `yo swiftserver`.
+
+-   IBM API Connect developer toolkit CLI: `apic swiftserver`.
+
+You use these command line tools to create and configure Swift Server Generator applications. The table below describes using the Yeoman command-line options. If you are familiar with the IBM API Connect developer toolkit and prefer to use that you can refer to the Swift [Server Generator installation and command line options](https://www.ibm.com/support/knowledgecenter/en/SSMNED\_5.0.0/com.ibm.apic.toolkit.doc/capic\_swift\_overview.html).

```

Comment:
https://www.ibm.com/support/knowledgecenter/en/SSMNED_5.0.0/com.ibm.apic.toolkit.doc/capic_swift_overview.html

Looks like this is the intended link. Pandoc has added some escaping here again.